templates/home/api.html

```diff
@@ -21,121 +21,147 @@
 	<dd>Template <!-- Common design patter --></dd>
 
 	<dt>Last review</dt>
-	<dd>2019-05-09</dd>
+	<dd>2022-04-27</dd>
 
 	<dt>Conforming to</dt>
-	<dd>Going to be: Content and IA spec 2.0</dd>
+	<dd>Going to be: Content and IA spec 3.0</dd>
 
 	<dt>Specification</dt>
-	<dd><a href="https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-information-architecture-specification/templates-detailed-specifications/home-page.html">Home page for Canada.ca</a>
+	<dd><a href="https://design.canada.ca/mandatory-templates/home-page.html">Home page for Canada.ca</a>
 </dl>
 
 <h2>Technical specification</h2>
 
-<h3>Home page banner</h3>
+<h3>Your government</h3>
 
 <dl>
 	<dt>Version</dt>
-	<dd>2</dd>
+	<dd>3.0</dd>
 	<dt>Updated</dt>
-	<dd>2019-05-09</dd>
+	<dd>2022-04-27</dd>
 </dl>
 
-<p>The background image is loaded with the provisional plugin bgimg which set the background image. The background image is hosted in the WMS at this location. <code>https://www.canada.ca/content/dam/canada/carousel/bkg-home-banner.jpg</code>.</p>
```

Comment:
This belong to the component "Home page banner", please keep it.

Reply:
Fixed. Thanks